src/main/scss/templates/_friends.scss

```diff
@@ -0,0 +1,53 @@
+// Friends styling
+.friends {
+  display: block;
+  padding: $size-base * 2;
+
+  .card {
+    background-color: $color-white;
```

Comment:
A lot of these styles are redundant. They're defined in _layout.scss's .card class. Please cross reference and remove the ones that are duplicates.